Gems/ROS2/Code/Include/ROS2/Lidar/LidarSystemBus.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/EntityId.h>
+#include <AzCore/EBus/EBus.h>
+
+namespace ROS2
+{
+    class LidarSystemRequests
+    {
+    public:
+        AZ_RTTI(LidarSystemRequests, "{007871d1-2783-4382-977b-558f436c54a5}");
+        virtual ~LidarSystemRequests() = default;
+
+        //! Creates a new Lidar.
+        //! @param lidarEntityId EntityId holding the ROS2LidarSensorComponent.
+        //! @return A unique Id of the newly created Lidar.
+        virtual AZ::Uuid CreateLidar(const AZ::EntityId& lidarEntityId) = 0;
```

Comment:
suggestion: Creating a typesafe version of `AZ::Uuid` might be a useful addition (could be outside the scope of this PR).

Reply:
I like Tom suggestion to do a typesafe version of `AZ::Uuid` for Lidar. Or as minimum an alias would be nice to make it easier to change it in the future.

````
using LidarId = AZ::Uuid;
````

That way `LidarId` could be used everywhere.

Reply:
Thanks! I've created the `LidarId` alias as you had suggested (inside the LidarRaycasterRequests class).

Reply:
This is definitely better but it's not type safe unfortunately (take a look at how we create type safe ids in the White Box Gem if you're interested (see WhiteBoxToolApi.h and look for all the handle types - e.g. VertexHandle, EdgeHandle etc... It makes it impossible to accidentally mix them up which is super useful (runtime bugs become compile time bugs 🥳). It uses this technique for reference - https://blog.demofox.org/2015/02/05/getting-strongly-typed-typedefs-using-phantom-types/.

Reply:
Thanks for this suggestion. I've updated this section of code. The references you provided were very helpful!

Reply:
No problem! Glad it was useful 🙂